modules/main/src/main/env/service/entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
This is the entrypoint.sh that will be used by the wasabi docker image for developer who want to build docker image for the current version of source code.  I will check in the dockerfile into a different branch but you want it in this branch that is fine with me.